tests/framework/validators/CompareValidatorTest.php

```
@@ -37,364 +43,836 @@ protected function tearDown(): void
     public function testValidateValueErrorContainsCompareValue(): void
     {
-        $val = new CompareValidator(['compareValue' => 'expected']);
-        $val->validate('wrong', $error);
-        $this->assertStringContainsString('expected', $error);
+        $validator = new CompareValidator(
+            [
+                'compareValue' => 'expected',
+            ],
+        );
+
+        $validator->validate('wrong', $error);
```

PHPMD (2.15.0)

[warning] Avoid using undefined variables such as '$error' which will lead to PHP notices. (undefined)

(UndefinedVariable)